src/components/__tests__/sidebar.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Get rid of this file and folder. FYI tests should live in the components folder alongside the components they're testing. (e.g. Sidebar.js - followed by - Sidebar.test.js)